debug.xml

```diff
@@ -320,6 +323,10 @@
         DELETE FROM notifications WHERE id = :id;
     </entry>
 
+    <entry key='database.clearPositionsHistory'>
+        DELETE FROM positions WHERE deviceid = :deviceId and servertime &lt; :serverTime;
```

Comment:
What happens to `positionId` in the device if it points to removed record? It should be handled.